tajo-core/src/test/java/org/apache/tajo/engine/query/TestSelectQuery.java

```diff
@@ -105,6 +105,20 @@ public final void testExplainSelect() throws Exception {
   }
 
   @Test
+  @SimpleTest(queries = {
```

Comment:
This still looks good, but will be great if the following two problems are fixed.
First, it seems to be difficult to execute tests individually. 
Second, when a query test fails, it will be difficult to figure out which query failed.
Do you have any good idea?